src/wazuh_db/wdb_integrity.c

```diff
@@ -638,3 +639,52 @@ int wdbi_check_sync_status(wdb_t *wdb, wdb_component_t component) {
     cJSON_Delete(j_sync_info);
     return result;
 }
+
+int wdb_get_global_group_hash(wdb_t * wdb, os_sha1 hexdigest) {
+    if (OS_SUCCESS == wdb_global_group_hash_cache_operations("read", hexdigest)) {
+        mdebug2("A global group hash was found in cache");
+        return OS_SUCCESS;
+    } else {
+        if(!wdb) {
+            mdebug1("Missing DB pointer to calculate the global group hash.");
+            return OS_INVALID;
+        }
+
+        sqlite3_stmt* stmt = wdb_init_stmt_in_cache(wdb, WDB_STMT_GLOBAL_GROUP_HASH_GET);
+        if (!stmt) {
+            return OS_INVALID;
+        }
+
+        int result = wdb_calculate_stmt_checksum(wdb, stmt, WDB_GENERIC_COMPONENT, hexdigest);
+        if(result == 1) {
```

Comment:
We can simplify this as:
`if (wdb_calculate_stmt_checksum(wdb, stmt, WDB_GENERIC_COMPONENT, hexdigest)) {`

Reply:
Done!